x/storage/keeper/keeper.go

```diff
@@ -641,22 +676,31 @@ func (k Keeper) SealObject(
 		return types.ErrObjectAlreadySealed
 	}
 
-	// check the signature of secondary sps
-	// SecondarySP signs the root hash(checksum) of all pieces stored on it, and needs to verify that the signature here.
-	var secondarySps []string
-	for i, spAddr := range opts.SecondarySpAddresses {
-		spAcc := sdk.MustAccAddressFromHex(spAddr)
-		secondarySps = append(secondarySps, spAcc.String())
-		sr := types.NewSecondarySpSignDoc(spAcc, objectInfo.Id, objectInfo.Checksums[i+1])
-		err := k.VerifySPAndSignature(ctx, spAcc, sr.GetSignBytes(), opts.SecondarySpSignatures[i])
-		if err != nil {
-			return err
-		}
+	gvg, found := k.virtualGroupKeeper.GetGVG(ctx, opts.GlobalVirtualGroupId)
```

Comment:
We need ensure that the `GlobalVirtualGroupId` serving a same bucket are under family.

Reply:
Fixed